distributions/otelcol-k8s/README.md

```diff
@@ -0,0 +1,18 @@
+# OpenTelemetry Collector Kubernetes Distro
+
+This distribution is made specifically to be used in a Kubernetes Cluster to monitor Kubernetes and services running in Kubernetes. It contains a subset of components from [OpenTelemetry Collector](https://github.com/open-telemetry/opentelemetry-collector) and [OpenTelemetry Collector Contrib](https://github.com/open-telemetry/opentelemetry-collector-contrib).
+
+## Components
+
+The full list of components is available in the [manifest](manifest.yaml)
+
+### Rules for Component Inclusion
+
+ - Only includes components from Contrib and Core.
+ - Only components that are Alpha or higher.
+ - All receivers must facilitate the collection of data that is generated by Kubernetes or services running in Kubernetes 
+   - Although you could run a collector in Kubernetes to reach out to any endpoint, this distribution is not targeting that use case. This distribution is for monitoring Kubernetes and the processes it runs. 
+ - All components must be vendor-neutral.
+ - Only exporters that use OTLP are allowed.
+   - The debug exporter is an exception.
+ - All technologies OTel promised to support will be included as receivers. Specifically this means the `jaegerreceiver`, `zipkinreceiver`, `opencensusreceiver`, and `prometheusreceiver` will be included.
```

Comment:
Given this is a new distribution, we don't need to keep this promise. I would remove Jaeger, OpenCensus, and Zipkin, keeping the door open to adding it in the future. If we add them now, it's harder to remove in the future.

Reply:
Based on https://github.com/open-telemetry/opentelemetry-collector-releases/issues/357#issuecomment-1593405517 I added those components.

Reply:
Would it be too schizophrenic to argue with myself? On Mar 2024, I would probably go with a smaller set of components, removing Zipkin/OpenCensus/Jaeger, leaving the door open to including them in the future if people need them.

@pavolloffay, we kind of talked about this at the Observatory last week, and I think you might have a different idea :-)

Reply:
I'm gonna argue that we keep them in for these reasons:
1. As far as I am aware our commitments to supporting interoperability with these projects has not changed.
2. OpenTelemetry has not graduated yet - there are plenty of people using jaeger/zipkin/opencensus in their k8s clusters.  Including these components lets users have a smoother transition to Otel.
3. The OpenTelemetry helm chart's default config depends on these components